project.php

```diff
@@ -466,29 +467,16 @@ function do_project_info_table()
     if (!empty($project->special_code))
     {
         $spec_code = $project->special_code;
-        if (
-            (strncmp($spec_code,'Birthday',8) == 0 ) or
-            (strncmp($spec_code,'Otherday',8) == 0 )
+        if (startswith($spec_code, 'Birthday') ||
+            startswith($spec_code, 'Otherday')
         )
         {
             $spec_display = $spec_code;
         }
         else
         {
-            $sql = sprintf("
-                SELECT display_name
-                FROM special_days
-                WHERE spec_code = '%s'
-            ", mysqli_real_escape_string(DPDatabase::get_connection(), $spec_code));
-            $spec_res = mysqli_fetch_assoc(mysqli_query(DPDatabase::get_connection(), $sql));
-            if ($spec_res)
-            {
-                $spec_display = $spec_res['display_name'];
-            }
-            else
-            {
-                $spec_display = "($spec_code)";
-            }
+            $special_days = load_special_days();
```

Comment:
You need to import `pinc/special_colors.inc` for this to work though.

Reply:
Argh! Sorry!